src/main.cpp

```diff
@@ -1,6 +1,6 @@
-#include <vector>
+#include "functions.hpp"
```

Comment:
It should be the last header here.

https://coders.school/edu/courses/powtorka-1-qa/lesson/kolejnosc-include/